src/coreComponents/constitutive/solid/Damage.cpp

```diff
@@ -143,16 +113,16 @@ void Damage< BASE >::postInputInitialization()
                                            " external driving force flag is turned on" );
 
   // set results as array default values
-  this->template getWrapper< array1d< real64 > >( viewKeyStruct::criticalFractureEnergyString() ).
+  this->template getWrapper< array1d< real64 > >( fields::solid::criticalFractureEnergy::key() ).
```

Comment:
Since we have done a `registerField`, I think we should also do a `getField` (if that works).
```suggestion
  getField< fields::solid::criticalFractureEnergy >( ).
```